library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -0,0 +1,415 @@
+            fill: orange;
+        }
+    }
+
+    .btn.add-rubric {
+        color: #0093b8;
+    }
+
+    .rubric-details {
+        padding: 10px;
+    }
+}
+
+/** External to the tool **/
+.sakai-rubric-association {
+    .disabled {
+        label {
+            color: #ccc;
+        }
+    }
+    .rubrics-list {
+        padding-left: 20px;
+    }
+    select.form-control {
+        -webkit-appearance: menulist;
+        width: auto;
+        display: inline-block;
+        margin: 0 20px 10px 0;
+        #rubricslist {
+            display: inline-block;
+        }
+    }
+}
+
+.rubrics-lightbox {
+    position: absolute;
+    z-index: 999;
+    background: rgba(0,0,0,0.5);
+
+    sakai-rubric-criterion-preview {
+        display: block;
+        padding-bottom: 20px;
+    }
+    .container {
+        background: #fff;
+        margin: 80px auto;
+        position: relative;
+        > a {
+            text-decoration: none;
+            padding: 1px 7px 6px;
+            font-weight: bold;
+            font-size: 18px;
+            background: #fff;
+            border: 1px solid #555;
+            border-radius: 30px;
+            display: block;
+            line-height: 1;
+            position: absolute;
+            top: -10px;
+            right: -10px;
+        }
+    }
+}
+
+.criterion {
+    &.sakai-rubric-criterion,
+    &.sakai-rubric-criterion-preview,
+    &.sakai-rubric-criterion-grading,
+    &.sakai-rubric-criterion-student,
+    &.sakai-rubric-criterion-readonly {
+        width: 100%;
+        border-bottom: 1px solid #d3d3d3;
+        .criterion-row {
+            width: 100%;
+            height: 100%;
+            display: table;
+            border: 1px solid #d3d3d3;
+            border-width: 1px 0 0 0;
+            &.sortable-ghost, .sortable-ghost {
+                opacity: 0.25;
+            }
+            &.sortable-drag, .sortable-drag {
+                cursor: grabbing;
+                cursor: -webkit-grabbing;
+                cursor: -moz-grabbing;
+            }
+            .criterion-detail {
+                position: relative;
+                padding: 10px;
+                display: table-cell;
+                vertical-align: top;
+                background: #f2f2f4;
+                width: 20%;
+                border-right: 1px solid #d3d3d3;
+                p {
+                    margin: 0;
+                    padding: 0;
+                    font-size: 11px;
+                    color: #414141;
+                }
+            }
+            .reorder-icon {
+                cursor: grab;
+                cursor: -webkit-grab;
+                cursor: -moz-grab;
+                &:active {
+                    cursor: grabbing;
+                    cursor: -webkit-grabbing;
+                    cursor: -moz-grabbing;
+                }
+            }
+            .criterion-ratings {
+                .rating-item {
+                    sakai-rubric-criterion-rating-edit {
```

Comment:
A few cases of potential class missing the starting period. (Unless it is a custom element)